docs/content/concepts/assets/multi-assets.mdx

```diff
@@ -28,109 +28,92 @@ The function responsible for computing the contents of any asset is an [op](/con
 
 ### A basic multi-asset
 
-The easiest way to create a multi-asset is with the <PyObject object="multi_asset" decorator /> decorator. This decorator functions similarly to the <PyObject object="asset" decorator /> decorator, but requires an `outs` parameter specifying each output asset of the function.
+The easiest way to create a multi-asset is with the <PyObject object="multi_asset" decorator /> decorator. This decorator functions similarly to the <PyObject object="asset" decorator /> decorator, but requires a `specs` parameter specifying each asset that the function materializes.
 
 ```python file=/concepts/assets/multi_assets.py startafter=start_basic_multi_asset endbefore=end_basic_multi_asset
-from dagster import AssetOut, multi_asset, AssetExecutionContext
+from dagster import AssetSpec, multi_asset
 
 
-@multi_asset(
-    outs={
-        "my_string_asset": AssetOut(),
-        "my_int_asset": AssetOut(),
-    }
-)
+@multi_asset(specs=[AssetSpec("users"), AssetSpec("orders")])
 def my_function():
-    return "abc", 123
+    # some code that writes out data to the users table and the orders table
+    ...
 ```
 
-By default, the names of the outputs will be used to form the asset keys of the multi-asset. The decorated function will be used to create the op for these assets and must emit an output for each of them. In this case, we can emit multiple outputs by returning a tuple of values, one for each asset.
-
 ### Conditional materialization
 
-In some cases, an asset may not need to be updated in storage each time the decorated function is executed. You can use the `is_required` parameter along with `yield` syntax to implement this behavior.
+In some cases, an asset may not need to be updated in storage each time the decorated function is executed. You can use the `skippable` parameter along with `yield` syntax and <PyObject object="MaterializeResult" /> to implement this behavior.
 
-If the `is_required` parameter is set to `False` on an output, and your function does not `yield` an <PyObject object="Output" /> object for that output, then no asset materialization event will be created, the I/O manager will not be invoked, downstream assets will not be materialized, and asset sensors monitoring the asset will not trigger.
+If the `skippable` parameter is set to `True` on an <PyObject object="AssetSpec" />, and your function does not `yield` a <PyObject object="MaterializeResult" /> object for that asset, then no asset materialization event will be created, downstream assets in the same run will not be materialized, and asset sensors monitoring the asset will not trigger.
```

Comment:
```suggestion
If the `skippable` parameter is set to `True` on an <PyObject object="AssetSpec" /> and your function does not `yield` a <PyObject object="MaterializeResult" /> object for that asset, then:

- No asset materialization event will be created, 
- Downstream assets in the same run will not be materialized, and 
- Asset sensors monitoring the asset will not trigger
```

I saw Jamie's comment about this paragraph being kinda dense. What d'you think about taking the opportunity to do something like this?

Reply:
👍